### DISCUSSION TITLE
Swift on z/OS

### DISCUSSION THREAD

**Discussion by fc3sunfire** [zopencommunity/meta#371] [Ideas]
I want to get a discussion about getting Swift on z/OS. Being Open Sourced for a few years now, I think we can bring more younger talent into the Mainframe space if we can get this ported. Let's get started!

**Comment by IgorTodorovskiIBM** [comment]
IBM introduced a Swift 4.x port for z/OS a while back (https://www.ibm.com/docs/en/SSMEGX_4.0.0/com.ibm.swift.zos.v4.doc/swift_zos_docs.pdf), but due to minimal interest, the project was eventually abandoned.

Swift made several attempts to venture into the backend space with frameworks like Kitura, but we didn't see much adoption in either on z/OS. Languages like Go and Rust seemed to have gained more traction, at least from my perspective. Do you see more back-end services being written in Swift these days?
